### ISSUE TITLE
attach binding on init bug

### ISSUE THREAD

**Issue by scarymrgrey** [Incoding-Software/Incoding-Framework#37]
When you add binding on initincoding (dsl.Self().Bind.Attach()) for 2 or more bindings ,it seems to rewrite previous bindings.

**Comment by IncodingSoftware** [comment]
Yes are you right . It will be fix but you can use multiple When now. 

Bind(dsl=>dsl.When(Click).When(Change))
